bower.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "ilios-calendar",
   "dependencies": {
-    "ember": "~2.8.0",
+    "ember": "~2.9.0-beta.1",
```

Comment:
We shouldn't be jumping up to 2.9.  I'm surprised it was in this release - may be an error on the ember-cli team.